synapse-modules/alkemio_room_control.py

```
@@ -239,63 +237,45 @@ def http_client(self) -> SimpleHttpClient:
-                    str(e),
-                )
-                return False
-        return False
+        try:
+            resp = await self.http_client.post_json_get_json(
+                check_url,
+                payload,
+                headers=headers,
+            )
+            return resp
+        except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)